fetch.bs

```diff
@@ -4523,6 +4527,22 @@ steps:
      <li><p>Return <var>fetchParams</var>'s <a for="fetch params">preloaded response candidate</a>.
     </ol>
 
+   <dt><var>request</var>'s <a for=request>service-workers race token</a> is not null
+   <dd>
+    <ol>
+     <li><p>Let <var>raceReesponse</var> to the result of [=lookup-race-response=] given <var>request</var>'s <a for=request>service-workers race token</a>, <var>request</var>'s <a for=request>reserved client</a>, <var>request</var>'s <a for=request>URL</a>.
```

Comment:
Will [=lookup-race-response=] be defined in SW spec?

Reply:
Also, I guess we only do the matching for HTTP/HTTPS and GET request.  Do we do such restriction here?

Reply:
> Will [=lookup-race-response=] be defined in SW spec?

Yes. It's added in https://github.com/yoshisatoyanagisawa/ServiceWorker/pull/10. I'm not sure if we have a good way to call an upcoming algorithm.

> Also, I guess we only do the matching for HTTP/HTTPS and GET request. Do we do such restriction here?

Those are described in https://github.com/yoshisatoyanagisawa/ServiceWorker/pull/10. The token is set under those restrictions. I guess we don't have to rephrase them.

Reply:
Thank you.